app/src/main/java/de/hhn/aib/labsw/blackmirror/controller/PageController.java

```diff
@@ -194,4 +202,74 @@ private void autoChangePageByTime() {
             }
         }
     }
+
+    @Override
+    public void dataReceived(String topic, JsonNode object) {
+        super.dataReceived(topic, object);
```

Comment:
This is redundant as `super.dataReceived()` is empty

Reply:
Done. Not sure where this came from. Wasn´t any longer on my local branch.